projects/metis/src/app/dataset/depublication/depublication.component.ts

```diff
@@ -310,37 +312,30 @@ export class DepublicationComponent extends DataPollingComponent {
   }
 
   /** onError
-  /* - run default error handlere
-  /* - set isSaving to false
-  /* - log the error to the console
-  /*  @param {HttpErrorResponse} err - the fail response
-  */
-  onError(err: HttpErrorResponse): void {
-    this.errors.handleError(err);
+   * - set isSaving to false and initialise notification
+   *  @param {HttpErrorResponse} error
+   */
+  onError(error: HttpErrorResponse): void {
     this.isSaving = false;
-    console.error(`${err.status}: ${err.statusText}`);
```

Comment:
Here as well I think can be usefull to keep the log

Reply:
It may be useful but it isn't best practice!

Reply:
It isn't best practice? Why do you say that?
Overall writing logs is a good practice since it gives a full view of the application's behavior which it helps tracing out errors/debugging when unexpected issues occur. For `info` level logs I can understand that we don't want to make the browser console noisy, but in this case for `error` type logs it is very useful. 

But from what you explained before, the method `httpErrorNotification` ensures that the error message is shown, so this is handled